test/getUriParameters-test.coffee

```diff
@@ -94,17 +107,44 @@ describe('Transformation • Refract • getUriParameters' , ->
             'type': 'enum',
```

Comment:
There is just one remaining problem with the type. In API Blueprint AST, the type is `number` and not `enum` in this case.

```
$ diff ../drafter/test/fixtures/api/resource-parameters.apib.ast.json ../drafter/test/fixtures/api/resource-parameters.apib.refract.json
31c31
<               "type": "number",
---
>               "type": "enum",
46c46
<               "type": "number",
---
>               "type": "enum",
```

#### Blueprint

```apib
+ Parameters
    + id: 23 (enum[number], optional) - Database ID

        Additional description

        + Default: 1

        + Members
            + 37 - Testing value
            + 1
            + 23
```

#### API Blueprint AST from Drafter

```yaml
          parameters:
            -
              name: "id"
              description: "Database ID\n\nAdditional description\n"
              type: "number"
              required: false
              default: "1"
              example: "23"
              values:
                -
                  value: "37"
                -
                  value: "1"
                -
                  value: "23"
```

#### Refract from Drafter

```yaml
                element: "hrefVariables"
                content:
                  -
                    element: "member"
                    meta:
                      description: "Database ID\n\nAdditional description\n"
                    attributes:
                      typeAttributes:
                        - "optional"
                    content:
                      key:
                        element: "string"
                        content: "id"
                      value:
                        element: "enum"
                        attributes:
                          samples:
                            -
                              -
                                element: "number"
                                content: 23
                          default:
                            -
                              element: "number"
                              content: 1
                        content:
                          -
                            element: "number"
                            content: 37
                          -
                            element: "number"
                            content: 1
                          -
                            element: "number"
                            content: 23
```

I'm not sure what we should do here. @pksunkara any ideas?